src/figuresClasses.ts

```diff
@@ -1,19 +1,78 @@
-export interface Figure {
+type Shape = 'triangle' | 'circle' | 'rectangle';
+type Color = 'red' | 'green' | 'blue';
 
+export interface Figure {
+  shape: Shape;
+  color: Color;
+  getArea(): number;
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  public shape: Shape = 'triangle';
```

Comment:
and afterwards, use it here, also redundant `public` modifier, fields are public by default, fix it everywhere
```suggestion
  shape: Shape = Shape.Triangle;
```